docs/develop/script-buttons.md

```diff
@@ -24,6 +24,10 @@ The following screenshot shows the script details page for a script titled **Cre
 
 To stop sharing a script through a button, go to the **More options (…)** menu in the script details page and select **Stop sharing**. This removes all the buttons that run the script. Deleting a single button removes the script from that one button, even if the operation is undone or the button is cut and pasted.
 
+## Older versions of Excel
+
+Script buttons won't work when opened in [versions of Excel that don't support Office Scripts](../testing/platform-limits.md#platform-support). In that case, the button still appears, but pressing it has no effect.
```

Comment:
According to [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29040) "press" is only for physical hardware. 
```suggestion
Script buttons won't work when opened in [versions of Excel that don't support Office Scripts](../testing/platform-limits.md#platform-support). In that case, the button still appears, but selecting it has no effect.
```